app/views/common/_navbar.html.erb

```diff
@@ -3,7 +3,10 @@
     <div class="container-fluid">
       <ul class="nav navbar-nav">
         <%= tab("Orders", orders_path, active == "orders") %>
-        <%= tab("Inventory", items_path, active == "inventory") %>
+
+        <% if current_user.super_admin? %>
```

Comment:
For consistency, I think this should be `if current_user.can_view_or_edit_items?`